mods/fire/init.lua

```diff
@@ -257,13 +267,8 @@ else
 		chance = 6,
 		catch_up = false,
 		action = function(p0, node, _, _)
-			-- If there are no flammable nodes around flame, remove flame
 			local p = minetest.find_node_near(p0, 1, {"group:flammable"})
-			if not p then
-				minetest.remove_node(p0)
-				return
-			end
-			if math.random(1, 4) == 1 then
+			if p and math.random(1, 4) == 1 then
```

Comment:
There's no point havng an ABM that essentially starts with:
```if math.random(1, 4) == 1 then```
As this is running 'find node near' 4 times more often than necessary.